ui/src/app/pages/homepage/data-access-requirements.spec.tsx

```diff
@@ -113,35 +135,36 @@ describe('DataAccessRequirements', () => {
         const activeModule = getActiveModule(enabledModules, testProfile);
 
         // update this if the order changes
-        expect(activeModule).toEqual(AccessModule.ERACOMMONS)
+        expect(activeModule).toEqual(AccessModule.RASLINKLOGINGOV)
 
         // 2FA (module 0) is complete, so enabled #1 is active
         expect(activeModule).toEqual(enabledModules[1]);
 
-        // eRA is the next module.
-        expect(activeModule).toEqual(allModules[1]);
-        // TODO(RW-7301): Revert this back.
-        // // but we skip allModules[1] because it's RAS and is not enabled
-        // expect(activeModule).toEqual(allModules[2]);
+        // but we skip allModules[1] because it's ERA and is not enabled
+        expect(activeModule).toEqual(allModules[2]);
     });
 
-    it('should return the second module (ERA) from getActiveModule when the first module (2FA) has been bypassed', () => {
+    it('should return the fourth module (Compliance) from getActiveModule when the first 3 modules have been completed', () => {
```

Comment:
this test was for an incorrect theory of the bug.  might as well keep it, though.